mkt/commonplace/templates/commonplace/index.html

```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html{% if appcache %} manifest="{{ url('commonplace.appcache')|urlparams(repo=repo) }}"{% endif %}>
+<html{% if appcache %} manifest="{{ url('commonplace.appcache')|urlparams(repo=repo) }}"{% endif %}{% if repo == 'fireplace' %} lang="{{ LANG }}" dir="{{ DIR }}"{% endif %}>
```

Comment:
I don't think so, I just don't think the other repos have the same list of available locales and I'd rather we not set this and then have it get changed once the page loads.